include/fe/inf_fe.h

```diff
@@ -71,21 +71,27 @@ class InfFERadial
    * \returns The first (local) derivative of the
    * decay in radial direction of the infinite element.
    *
-   * This is only valid for 3D?? - RHS
+   * This is only valid for 3D!
```

Comment:
So ... could we get any kind of protection on this (and later)?  Require an ```Elem*``` and assert (then eventually switch) based on elem->dim()?  Require base_elem to already be built and check that?

We laugh at the old joke about the dimwitted new programmer who slowly realizes, "I don't think the compiler even *reads* the comments!", but frankly this dimwitted library developer is slowly realizing that you can't count on the app developers reading them all either.

Reply:
you are right; after setting `decay` to valid values in all dimensions, we should adjust the derivative...